pkgs/applications/video/webtorrent_desktop/node-packages.nix

```diff
@@ -0,0 +1,4386 @@
+    "junk-3.1.0" = {
+      name = "junk";
+      packageName = "junk";
+      version = "3.1.0";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/junk/-/junk-3.1.0.tgz";
+        sha512 = "pBxcB3LFc8QVgdggvZWyeys+hnrNWg4OcZIU/1X59k5jQdLBlCsYGRQaz234SqoRLTCgMH00fY0xRJH+F9METQ==";
+      };
+    };
+    "k-bucket-5.0.0" = {
+      name = "k-bucket";
+      packageName = "k-bucket";
+      version = "5.0.0";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/k-bucket/-/k-bucket-5.0.0.tgz";
+        sha512 = "r/q+wV/Kde62/tk+rqyttEJn6h0jR7x+incdMVSYTqK73zVxVrzJa70kJL49cIKen8XjIgUZKSvk8ktnrQbK4w==";
+      };
+    };
+    "k-rpc-5.1.0" = {
+      name = "k-rpc";
+      packageName = "k-rpc";
+      version = "5.1.0";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/k-rpc/-/k-rpc-5.1.0.tgz";
+        sha512 = "FGc+n70Hcjoa/X2JTwP+jMIOpBz+pkRffHnSl9yrYiwUxg3FIgD50+u1ePfJUOnRCnx6pbjmVk5aAeB1wIijuQ==";
+      };
+    };
+    "k-rpc-socket-1.11.1" = {
+      name = "k-rpc-socket";
+      packageName = "k-rpc-socket";
+      version = "1.11.1";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/k-rpc-socket/-/k-rpc-socket-1.11.1.tgz";
+        sha512 = "8xtA8oqbZ6v1Niryp2/g4GxW16EQh5MvrUylQoOG+zcrDff5CKttON2XUXvMwlIHq4/2zfPVFiinAccJ+WhxoA==";
+      };
+    };
+    "keycode-2.2.0" = {
+      name = "keycode";
+      packageName = "keycode";
+      version = "2.2.0";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/keycode/-/keycode-2.2.0.tgz";
+        sha1 = "3d0af56dc7b8b8e5cba8d0a97f107204eec22b04";
+      };
+    };
+    "keypress-0.2.1" = {
+      name = "keypress";
+      packageName = "keypress";
+      version = "0.2.1";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/keypress/-/keypress-0.2.1.tgz";
+        sha1 = "1e80454250018dbad4c3fe94497d6e67b6269c77";
+      };
+    };
+    "languagedetect-1.2.0" = {
+      name = "languagedetect";
+      packageName = "languagedetect";
+      version = "1.2.0";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/languagedetect/-/languagedetect-1.2.0.tgz";
+        sha512 = "BMs1BIWCNcGtLrHA8zDBNnZzAMnYy7hrrU2F6yDhNScQ4q9sySy2RXEMZirQFJ8c8nAtLFwTWDxwjj97CG1LHQ==";
+      };
+    };
+    "last-one-wins-1.0.4" = {
+      name = "last-one-wins";
+      packageName = "last-one-wins";
+      version = "1.0.4";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/last-one-wins/-/last-one-wins-1.0.4.tgz";
+        sha1 = "c1bfd0cbcb46790ec9156b8d1aee8fcb86cda22a";
+      };
+    };
+    "load-ip-set-2.1.0" = {
+      name = "load-ip-set";
+      packageName = "load-ip-set";
+      version = "2.1.0";
+      src = fetchurl {
+        url = "https://registry.npmjs.org/load-ip-set/-/load-ip-set-2.1.0.tgz";
+        sha512 = "taz7U6B+F7Zq90dfIKwqsB1CrFKelSEmMGC68OUqem8Cgd1QZygQBYb2Fk9i6muBSfH4xwF/Pjt4KKlAdOyWZw==";
+      };
+    };
+    "load-json-file-1.1.0" = {
+      name = "load-jso
```

Comment:
This is wrong. Let me check how to embed the fetch source here.

Reply:
renders wront :-/
this is the line https://github.com/NixOS/nixpkgs/pull/74056/files/c8c8627fa36fddd5a3c2a106b5f2f0d5b592f0fb#diff-620905878edda2a966e61192c9004855R3777